analyzers/src/SonarAnalyzer.Common/Rules/CertificateValidationCheckBase.cs

```diff
@@ -308,7 +308,10 @@ private static ImmutableArray<TInvocationExpressionSyntax> FindInvocationList(Sy
             var ret = ImmutableArray.CreateBuilder<TInvocationExpressionSyntax>();
             foreach (var invocation in root.DescendantNodesAndSelf().OfType<TInvocationExpressionSyntax>())
             {
-                if (c.SemanticModel.GetSymbolInfo(invocation).Symbol.Equals(method))
+                if (Language.Syntax.InvocationIdentifier(invocation) is { } invocationIdentifier
+                    && invocationIdentifier.ValueText.Equals(method.Name)
```

Comment:
We'll need to use the `Language.NameComparison` here and add VB.NET test case with different casing.